code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -138,3 +138,6 @@
 
 /obj/item/encryptionkey/secbot
 	channels = list(RADIO_CHANNEL_AI_PRIVATE = 1, RADIO_CHANNEL_SECURITY = 1)
+
+/obj/item/encryptionkey/strandcuffs
+	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_MEDICAL = 1)
```

Comment:
wouldn't this make anyone holding the cuffs, able to hear medical comms?

Reply:
Not from my testing. Do you hear medcomms when standing over to a medborg?

Reply:
I can double-test again, I tested for general comms but I hadn't done specific channels.

Reply:
> Not from my testing. Do you hear medcomms when standing over to a medborg?

yea you do, medbots too